rust/utils/run-locally/src/utils.rs

```diff
@@ -27,62 +40,62 @@ pub fn concat_path(p1: impl AsRef<Path>, p2: impl AsRef<Path>) -> PathBuf {
     p
 }
 
-pub type AgentHandles = (Child, TaskHandle<()>, TaskHandle<()>);
+pub trait ArbitraryData: Send + Sync + 'static {}
+impl<T: Send + Sync + 'static> ArbitraryData for T {}
+
+pub type AgentHandles = (
```

Comment:
don't feel strongly but wonder if this were a struct if it'd be easier to work with in the future

Reply:
Maybe, but right now it would add more code.